### PR TITLE
[eclipse/xtext#2042] package old generator separately

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext#2042] package old generator separately

### CODE REVIEW

File: releng/org.eclipse.xtext.generator.feature/feature.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+         install-size="0"
+         version="0.0.0"
+         unpack="false"/>
+   <plugin
+         id="org.eclipse.xtend.typesystem.emf"
+         download-size="0"
+         install-size="0"
+         version="0.0.0"
+         unpack="false"/>
+   <plugin
```
Comment (cdietrich) [inline releng/org.eclipse.xtext.generator.feature/feature.xml:77]: should we package them or require them and package via catgegory.xml?

Comment (cdietrich) [reply releng/org.eclipse.xtext.generator.feature/feature.xml:77]: https://github.com/eclipse/xtext-umbrella/compare/cd_moreOldGeneratorExperiments_variant_b

Comment (kthoms) [reply releng/org.eclipse.xtext.generator.feature/feature.xml:77]: I prefer category.xml. There we have "3rd party" deps declared which constituate to the p2 repo and make the feature resolvable. The feature should only include own bundles.

**Comment by cdietrich** [comment]
abandoned in favour of https://github.com/eclipse/xtext-umbrella/pull/303

**Pull request closed without merging**